src/app/ui/superfest/SuperfestMissionPage.tsx

```diff
@@ -0,0 +1,8 @@
+'use client';
+import { SuperfestMissionPage } from 'src/components/Superfest/SuperfestPage/SuperfestMisisonPage';
+
+const SuperfestPage = ({ quest, url }: any) => {
+  return <SuperfestMissionPage quest={quest} baseUrl={url} />;
+};
+
```

Comment:
This seems to be a component which might not be needed? Can we not directly use `SuperfestMissionPage` without this `SuperfestPage`?

Reply:
I will.